proxy/http/HttpTransact.h

```diff
@@ -948,6 +948,9 @@ class HttpTransact
       }
       internal_msg_buffer_size = 0;
     }
+
```

Comment:
I am unclear on why you need to keep a copy of the proxy protocol information in the HttpState object.  Is this for future growth.  I don't see where this PR makes use of that data.

Reply:
The Proxy Protocol info is detected, parsed and stashed into the NetVC object. It is then copied into the State object in HttpTransact::initialize_state_variables_from_request() so that it can be accessed in HttpTransactHeaders::add_forwarded_field_to_request to write the Forwarded: header.  Somehow, the changes in HttpTransactHeaders.cc didn't get committed into this PR. Will fix this.

Reply:
Sorry, I had forgotten about a late change in the code.  Instead of waiting until it's time to write the headers out, I use the address received in the Proxy Protocol info and set it as the remote address in NetVConnection::get_remote_addr().  This way all internal references to the remote address use the info from Proxy Protocol.
I do store all of the data received in the Proxy Protocol header, and hope that someday, we can use that data in further enhancements, like a full Proxy Protocol implementation.